src/in_game/in_game.ts

```diff
@@ -73,42 +70,31 @@ class InGame extends AppWindow {
 
       return false;
     });
-    this.logLine(this._eventsLog, e, shouldHighlight);
-  }
-
-  // Displays the toggle minimize/restore hotkey in the window header
-  private async setToggleHotkeyText() {
-    const gameClassId = await this.getCurrentGameClassId();
-    const hotkeyText = await OWHotkeys.getHotkeyText(
-      kHotkeys.toggle,
-      gameClassId
-    );
-    const hotkeyElem = document.getElementById('hotkey');
-    hotkeyElem.textContent = hotkeyText;
   }
 
-  // Sets toggleInGameWindow as the behavior for the Ctrl+F hotkey
-  private async setToggleHotkeyBehavior() {
-    const toggleInGameWindow = async (
-      hotkeyResult: overwolf.settings.hotkeys.OnPressedEvent
-    ): Promise<void> => {
-      console.log(`pressed hotkey for ${hotkeyResult.name}`);
-      const inGameState = await this.getWindowState();
-
+  // Sets handleTabHold to when 'Tab' is in the state 'down' the overlay is displayed
+  private async handleTabHold(hotkeyEvent) {
+    console.log(`state = ${hotkeyEvent.state}`);
```

Comment:
These log lines can be removed. They were added to make sure the feature worked as expected. (all three)